### ISSUE TITLE
Feature request: Add comments

### ISSUE THREAD

**Issue by Zeioth** [gera2ld/markmap#98]
### Markdown syntax
![screenshot_2022-01-04_14-27-21_935202807](https://user-images.githubusercontent.com/3357792/148065952-bacdcadb-4cef-417b-a4ef-e903ce87ddce.png)

### Rendered version:
Shows comment in a balloon on mouseover/click an item

**Comment by gera2ld** [comment]
Is this a standard syntax? If not, I prefer using a comment, for example:

```md
* item 1 <!-- tell markmap to show content below in a balloon -->

  description

* item 2 <!-- tell markmap to show content below in a balloon -->

  description
```

**Comment by Zeioth** [comment]
I agree with your syntax. That way we could comment any kind of node (h1-h5 headers, and orderes/unordered list items)

The main idea is be to have something similar to this.

![screenshot_2022-01-13_19-10-22_872226863](https://user-images.githubusercontent.com/3357792/149385556-869aea8b-c364-47a5-ac11-5ee56d2251b2.png)

Just a KISS balloon displaying comments about the node.
